app-shared-tests/src/main/java/com/appunite/loudius/AbsReviewersScreenTest.kt

```diff
@@ -99,6 +118,13 @@ abstract class AbsReviewersScreenTest {
             composeTestRule
                 .onNodeWithText("Uh-oh, it seems that Loudius has taken a vacation. Don't worry, we're sending a postcard to bring it back ASAP!")
                 .assertIsDisplayed()
+
+            expectThat(analyticsRule.analytics.log).containsExactly(
+                AnalyticsLog("action_start", mapOf("item_name" to "fetch_reviewers_data")),
+                AnalyticsLog("action_finished", mapOf("item_name" to "fetch_reviewers_data", "success" to true)),
+                AnalyticsLog("button_click", mapOf("item_name" to "notify")),
+                AnalyticsLog("action_finished", mapOf("item_name" to "notify", "success" to false))
```

Comment:
question: shouldn't be there also:
```kotlin
AnalyticsLog("action_start", mapOf("item_name" to "notify")),
```
?

Reply:
Good point, because we don't have `action_start` for this `action_finished` ;) I will implement it.